frontend/src/app/manage-subscription/page.tsx

```
@@ -439,451 +495,351 @@ export default function ManageSubscriptionPage() {
             </div>
           </motion.div>
-        </motion.div>
+        )}
+
+        {/* Footer */}
+        <div className="flex items-center justify-end pt-6">
+          <button
+            onClick={handleCancelSubscription}
+            className="flex items-center gap-2 text-sm text-red-600 dark:text-red-400 hover:text-red-700 dark:hover:text-red-300"
+          >
+            <X className="w-4 h-4" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)